frontend/layouts/default.vue

```diff
@@ -108,6 +105,9 @@ export default Vue.extend({
         setWalletModal(open: boolean): void {
             this.$store.commit('modals/setWalletModal', open);
         },
+        hasAcceptedTerms(): boolean {
+            return this.$cookies.get('terms-and-conditions') === 'accepted';
+        },
```

Comment:
Why did you change it to be a method instead of a computed function? Making it a method means that it wouldn't be reactive.

Reply:
Setting the cookie didn't seem to automatically update the computed property, so I had to reload the for it to change. That's why I changed it to check on button press with a method.

Is there maybe a way to "force" an update?

Reply:
Forcing an update is not a correct solution, [the variable should be reactive in the first place for the update to happen](https://github.com/tofandel/vue-cookies#readme). See my comment below.